examples/VIAME_OcclusionSaliency.ipynb

```diff
@@ -0,0 +1,504 @@
UE07wIQlFyKqewX/lgm5eUnuCwKTNF7x8UD3IJw3zEuIznHSgMmS/ZxMMA8Zc3S8WRyc+uWKMGSSIL4Ze9ZMHIy/yiXDQGw0E+xM0/ITn5iiY/3ibvVr2UL3uNmMGImciPSGqZZwIsxH63yOQF2ebSmV7vqob29vaV7htWh0xbl8PK//uu/np+fM34Gz1YKhwdEXjN4r6+vLy8vydsi/vCm1Kb8wPKrpE6m+w0S2rz7f1OzLExO8kZ0dsxqNQ+jf3p6ent768OPgpNwRze2idgCtOxR5K++rwY/oQQJRFTmXXOME7SA9ystgoGD8qWmm0SuKUtoux0Kyr/upyYWrBezG5zvEYnIIknnICQwr1sGEGEZfOEk5FYlPDi0K6StBN2+NZS7+fpKIci4wffe9OL0L3VKOPeEtOl0swRXRiJ/qwjd3mJmodyoNfraV1wJfPkWb7A6zXluoo98dCKdZIxMG/LM0dlj3hLZyaD3gchIPcC/H/XvUqaPMWwiTksQLed9H9EIY1un9nwjoauY1T3XXPXVxnodKp8jUZ/xH4TSUuAwwJlBkAdfM/Ab5MvKLf/q3hQPTWip0/Wkaen++vVLUg0BBg+8DJenNObTO9G+z7t0keUP5MYqrMzAu1RcCbrNvIPrwWI+s644SslTahmmOjywL+FdlBGHq6ISpzWWAJeN5CkJfSxeuV/RqhObsiNO8pJz0VmQirjYSxswL2YBsrHT0dOltQPnTWwejwUIA4ot7fPSbNMHBM4x0hO4Izzw+eqO2xLhKxQ12gpYznJrTR05123ebqkvef6YdMD5YZZl7jWMiOfn58vlwmv66XXw3Szc71ntcobMxX6SFeS4SSxt/aCJ9E+4X6GPjYR6fHzMeNiqApdcG8xs+5OyJpAaka0h/zjoxghAGEMh7U/OuSoRSrEo42WOWhEJNyhibmME5JuMvnQ6RpWhcnMEBRooN1NhcSxJC4dbZOULHbLMbttlPp01ZpuD1Z4E/sVh67asx5+zsvT3wBteZgVc87gvji4Mc/NVqQa6pU8LzLkc10e0TORas+3IQZhsdLQAy3quwAhPLZL7G4PWuFwiXyVLH6MPcFWZO0LWwFXLA/c7rfr19h0eV1GbslLhQhUx+8zSF1MmvmU8XwXD4ZF2NqU0Sq5+6F+txEX5FaJCZfgmZldh41+tCgsKj86els8PeyXWSsc97TKmsAJpL1elJheZU3S6akd0gWuZTXcx1ovwBuHjlPOt8pxhb7nN0Ob8q/w8g6Fcv82lTCjAmzRaAHA6PLx/jBnqqtBAffbwo5rcMCqKiDy/Qe7CLaUBTGfGuc9m7kARyuThr0bBMyir3CUSeJvJqjh4LirLwcrflMxhi4Te40prrxDCL1EZZ2mU9fvwZ3oVpJCS9zNvTAIehTxjgwPHkkRxw42XFkDPNwqNEhIW5o5SAaNVu+gRUABKv337hsp5wWwMocH7dNioQGsrGJg8UCcdFlhl8FNckG40tbtzJdU9Pj7mgZbv7+/uKGa5T0CRwhdzkEb6YIxvuj8sv5aHBIo5x+ZU6EVm8vM8kfqYf3GoiUwGqPCY08fLUZyUu4UTAkRxulXHPQIJyUX58PLywi1ky6Urzz2eY2Ufwpqt0qTO3jsWNGDHlyuH+SHrV7Zor8rcWNsKmCDuVxYPPR6DNsXIOI4DDupGAqb7ymfbZPn8/GSbmG1Q9J67hvp83yD3lWTw+vqanzBdQYdhFehuvroQWJXVmIP2mCFExBs2MDWYKYacx8hXCYEeGiUIXRWdze6PZgcVPhvQAsjbnPIepqXw8hwnBjFfMOHxy2bJJ9w7a4hjnH2CFnAiM7DB6p+BdEXO5RYF7IqkNBB9yDETocen8W0+rPjFByLltH2Mq/qbZvGvk9+q65JVE3hIs5Xd4M222dmWNvEG8nBVuYSNW2AwWEGgWfHE8BXok4mZO+VQjuwCYLUlMEhT55wfiJdxrXidW76S+p7/w8XlTB9xrovd99jnUDF/uyI2TLG0I9+mBkEddp7Bm9KUQN6GOchhcvbHMqiXJPHw8CBLDYKB085b5yFnaDj3SgTROcv84zjQnbCJOGYHbft3Holq/TN/gp39+Pi40heIY3ZyiRl7jUIVJx2EFsZcjAF5ic9FH4kMZ/JI4F1UMmZ4Rkrp4RS4YkOh5KyGwWLv4l6hODtIHVYWpeCV+jGHYKR+/puXnglC2mxaOB5iRBxY6cdgCnAxguDHyE7ECl7Y1nfhXH5wIg+acQkfMwuDcvq8InfSMh28tbe3t5eXl7///vvp6Un21GTjiDiD4B3hmaoqeZJZARewOS19fHz4AGU4PHA8s8XYRli8W8C9947sSha8TBL3ESErPzGzZQlf/6MV89XYrhYhkZjRgWu7cREwviqbYkhYDawmQ8DAAzyZdIMZrTjHfz/YMhD+8fHx7du3y7hlG13nzQF+mqrIzdWUnZTBvIGq/JyPDwmTFCtKYOZktPDeP188hHObfrKDysA4N3aLI27IAJsJ5X2js4+wZJ/RTRdPnvXaKLSJsNY5r9ehMozgUiFxs39WbqFtxtiG5ZnSWCv3sRwnRYAXbd2GX3qOdKA2zpVKQQwcpt/Oep29IB+jT2gYGfBwHF1ckIjP5ITP/dztDWDSLxQSQ+6rQGxQ7uWJAy/uolg5LuFFPAYltTLMEpLbgITiVtSNJYOpMlmrFso5XZkRMA68NRYj53r/sJMcY6askITUyCBmAErzbs+88q2Uq8t0UVHpqpdOqUQb9hEGLHmwk5UvnBWGf/5JDiHolU3MWD2rrGAXVm2szh3HARv3HEeAsC8hK3Lo3Vmpj1suoTgOyty7RbCXoiy2k9WHfXxQju6+vjcoQKbIBaqdGYEEdgNiSO9sJJVg8nKaN4jSrvROGwFuBE1WQh4weB7p7UWk31Ftqt982Co3JsgHli7+GLZSqvCk8PSFcUqWi9270yp3lGm+zQ6tk0EJBteHeOyLhMN8yDsVkzzcvjpoV0Ua/
```

Comment:
Is there a difference with this function compared to the other notebooks? If not, we may want to break this function out into a helper module here in the `examples` dir for use across the multiple notebooks that are utilizing it.

Reply:
It's largely the same, but there are small differences in how bioharn/netharn requires inputs for its predict function (i.e. a list of numpy arrays). I would push for getting this notebook up first, and we can discuss whether pulling out that function makes sense in the future.

Reply:
agreed :+1: